README.md

```diff
@@ -3,13 +3,18 @@ to FreeIPA web interface.
 
 To use it, do this:
 * BACKUP /etc/httpd/alias to some safe place (it contains private keys!)
-* clone/unpack all scripts including "ca" subdirectory somewhere
-* set WORKDIR and EMAIL variables in scripts setup-le.sh and renew-le.sh
+* clone/unpack all scripts including "ca" subdirectory somewhere (/root/ipa-le is the default)
+* set WORKDIR variable to the directory you cloned the repository to in scripts setup-le.sh and renew-le.sh
+* set EMAIL variable in script renew-le.sh
+* run "yum install dnf" (a stock FreeIPA machine doesn't have dnf installed)
```

Comment:
Fedora has dnf by default. Perhaps this could be changed to "make sure you have dnf installed"?